docker/gpu/dockerfile.gpu

```diff
@@ -73,8 +77,8 @@ RUN echo "export PATH=$CONDA_DIR/bin:"'$PATH' > /etc/profile.d/conda.sh && \
     /bin/bash ~/miniforge.sh -b -p $CONDA_DIR && \
     rm ~/miniforge.sh
 
-RUN conda config --set always_yes yes --set changeps1 no && \
-    conda create -y -q -n py3 numpy scipy scikit-learn jupyter notebook ipython pandas matplotlib
```

Comment:
Could you please clarify why these changes were needed?

Reply:
Hello, 

I personally thought it would be good to add this because if any error is thrown, you previously could not see what is going on, or where the script is stuck (as it was hidden), so I have made the logs during conda runs visible :) 

Thank you for reviewing the merge request, and let me know if I should change this line back to the original line.

Reply:
Thanks for the clarification!

Please revert changes in lines 76-77 because they are unrelated to the issue this PR is solving.

If you think that it's better to make conda installation more verbose, please open **a separate PR** and make corresponding changes in all Docker files in this repo:
https://github.com/microsoft/LightGBM/blob/83627ff0d6baf49e3b18991f47b12d9fa5724cbc/docker/dockerfile-python#L19-L21
https://github.com/microsoft/LightGBM/blob/83627ff0d6baf49e3b18991f47b12d9fa5724cbc/docker/gpu/dockerfile.gpu#L76-L77

Reply:
I am so sorry for the delay, the change has been made, opening the other Pull Request now :) @StrikerRUS